Asterix/utils/save_and_read.py

```diff
@@ -39,6 +39,9 @@ def save_plane_in_fits(dir_save_fits, name_plane, image):
     current_time_str = datetime.datetime.today().strftime('%H_%M_%S_%f')[:-3]
     name_fits = current_time_str + '_' + name_plane
 
+    if os.path.exists(dir_save_fits) is False:
+        raise Exception("Please define an existing directory for dir_save_fits keyword or None")
```

Comment:
Instead of a bare exception, a `FileNotFoundError` would be much more appropriate here.

Reply:
ok I agree. I will do in the other branch, because I have modified already all "if os.path.exists(xxx) is False:" to "if os.path.exists(xxx)" so it will produce stupid conflictx everywhere.